soc/riscv/openisa_rv32m1/Kconfig.soc

```diff
@@ -18,6 +18,9 @@ config SOC_OPENISA_RV32M1_RISCV32
 	select HAS_RV32M1_FTFX
 	select HAS_FLASH_LOAD_OFFSET
 	select BUILD_OUTPUT_HEX
+	select RISCV_ISA_RV32I
+	select RISCV_ISA_EXT_M
+	select RISCV_ISA_EXT_A
```

Comment:
Ok, docs for these were harder to find, so I may be even more off, but it seems that it should have C but no A, no? https://github.com/lowRISC/ibex and https://github.com/openhwgroup/cv32e40p/blob/master/docs/source/intro.rst

Reply:
ditto